elements/pfe-clipboard/src/pfe-clipboard.js

```diff
@@ -0,0 +1,173 @@
+import PFElement from "../../pfelement/dist/pfelement.js";
+
+class PfeClipboard extends PFElement {
+  static get tag() {
+    return "pfe-clipboard";
+  }
+
+  static get meta() {
+    return {
+      title: "Clipboard",
+      description: "Copy current URL to clipboard."
+    };
+  }
+
+  get templateUrl() {
+    return "pfe-clipboard.html";
+  }
+
+  get styleUrl() {
+    return "pfe-clipboard.scss";
+  }
+
+  static get events() {
+    return {
+      copied: `${this.tag}:copied`
+    };
+  }
+
+  // Declare the type of this component
+  static get PfeType() {
+    return PFElement.PfeTypes.Content;
+  }
+
+  static get properties() {
+    return {
+      iconHidden: {
+        title: "Icon Hidden",
+        type: Boolean,
+        attr: "icon-hidden"
```

Comment:
You don't need to set the attribute name unless it differs from the property.  Because it's `iconHidden`, it will automatically use `icon-hidden` for the property name.